frame/staking-rewards/src/lib.rs

```diff
@@ -295,4 +450,21 @@ pub mod pallet {
 			Err("Not implemented".into())
 		}
 	}
+
+	impl<T: Config> Pallet<T> {
+		pub(crate) fn pool_account_id(pool_id: &T::RewardPoolId) -> T::AccountId {
+			T::PalletId::get().into_sub_account(("po", pool_id))
+		}
+
+		pub(crate) fn reward_multiplier(
+			rewards_pool: &RewardPoolOf<T>,
+			duration_preset: DurationSeconds,
+		) -> Option<Perbill> {
+			rewards_pool.lock.duration_presets.get(&duration_preset).cloned()
+		}
+
+		pub(crate) fn boosted_amount(reward_multiplier: Perbill, amount: T::Balance) -> T::Balance {
```

Comment:
Do we need a specific method for this?

Reply:
I created it to we can use it for tests.